modules/upgrade/pages/upgrade-script-example.adoc

```diff
@@ -62,3 +66,296 @@ Use this script as the basis for an upgrade from SLE{nbsp}12{nbsp}SP4 to SLE{nbs
   </scripts>
 </profile>
 ----
+
```

Comment:
The URLs in this profile are wrong. The last part of the path must be the same as it is the distribution label.

`summer2017` in a mix with `autumn2018` cannot work.

I would suggest we unify all example profiles and use a variable `$distrotree` like in the SAP example.